node/test/benchmark/index.js

```diff
@@ -15,6 +16,7 @@ module.exports = async (options = {}) => {
   const allUseCasesConfig = new Map([
     ...Array.from(await resolveUseCasesConfig.node(coreConfig, options)),
     ...Array.from(await resolveUseCasesConfig.python(coreConfig, options)),
+    ...Array.from(await resolveUseCasesConfig.go(coreConfig, options)),
```

Comment:
Will it just work on my machine? Or should I have something preinstalled so it works as inspected?

e.g. in Node.js and Python cases, we expect the dependencies in their packages to be installed (they're not auto installed by benchmark script). I'm not sure how eventual dependencies are resolved in case of Go

Reply:
The only thing you need is [Go to be installed on your machine](https://go.dev/dl/). [`go build` command](https://github.com/serverless/console/blob/e0d2f7f67a3aa7b565673448114c017698b7363a/go/packages/slslambda/scripts/build-go-binary.sh#L2) will fetch all required dependencies automatically.